apps/web/src/components/auth/CreateOrganizationForm.tsx

```diff
@@ -77,6 +87,11 @@ export function CreateOrganization({}: Props) {
     }
 
     setLoading(false);
+    if (isFromVercel) {
+      startVercelSetup();
+
+      return;
+    }
     navigate('/quickstart');
```

Comment:
I would simplify these lines into `isFromVercel ? startVercelSetup() : navigate('/quickstart')`

Reply:
I think this is fine because it is easier to read even if it makes more code.